### PR TITLE
Update to the license headers

### PR DESCRIPTION
**Pull request by imphil**
These changes have been approved by Luca and Frank. Review is needed for style and other non-content topics.

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Pirmin Vogel
+- Rahul Behl
+- Rhys Thomas
+- Renzo Andri
+- Robert Schilling
+- Scott Johnson
+- Stefan Wallentowitz
+- Sven Stucki
+- Tao Liu
+- Tobias Wölfel
```
[the remark this answers is not in the document]
Comment (vogelpi) [reply CREDITS.md]: Should also "Udi Jonnalagadda" be added?

**Comment by imphil** [comment]
Thanks @vogelpi for your review, I've updated the text accordingly.

Regarding more explicit rules on what gets your name into CREDITS.md: I wanted to make the text as appreciative and thankful as we can be. Since there is no automatism in place to get a name added to the CREDITS.md file, we can always decide on a case-by-case basis if we feel that it is abused in any way.

**Comment by vogelpi** [comment]
Okay, @imphil this sounds good to me!

File: README.md [hunk 2]
```diff
@@ -57,6 +57,11 @@ Zulip](https://lowrisc.zulipchat.com/#narrow/stream/198227-ibex)!
 Unless otherwise noted, everything in this repository is covered by the Apache
 License, Version 2.0 (see LICENSE for full text).
 
+## Credits
+
+Many people have contributed to Ibex through the years. Please have a look at
+the [credits file](CREDITS.md) for more information.
```
Comment (asb) [inline README.md]: `the [credits file](CREDITS.md) and the commit history for more information.`

Comment (imphil) [reply README.md]: updated.

**Pull request merged**